wg-api/spec-documents/scroll-view.md

```diff
@@ -0,0 +1,256 @@
+
+#### `scrollView.setContentSize(size)`
+
+Set the size of the contents.
+
+#### `scrollView.getContentSize()`
+
+Returns the `size` of the contents.
+
+#### `scrollView.setHorizontalScrollBarMode(mode)`
+
+* `mode` string - Can be one of the following values: `disabled`, `hidden-but-enabled`, `enabled`. Default is `enabled`.
+
+Controls how the horizontal scroll bar appears and functions.
+* `disable` - The scrollbar is hidden.
+* `hidden-but-enabled` - The scrollbar is hidden whether or not the contents are larger than the viewport, but the pane will respond to scroll events. This option is not available on MacOS.
+*`enabled` - The scrollbar will be visible if the contents are larger than the viewport.
+
+#### `scrollView.setVerticalScrollBarMode(mode)`
+
+Controls how the vertical scroll bar appears and functions.
+
+#### `getHorizontalScrollBarMode()`
+
+Returns `string` - horizontal scrollbar mode.
+
+#### `scrollView.getVerticalScrollBarMode()`
+
+Returns `string` - vertical scrollbar mode.
+
+#### `scrollView.setHorizontalScrollElasticity(elasticity)` _macOS_
+
+* `elasticity` string - Can be one of the following values: `automatic`, `none`, `allowed`. Default is `automatic`.
+
+The scroll view’s horizontal scrolling elasticity mode.
+A scroll view can scroll its contents past its bounds to achieve an elastic effect. 
+When set to `automatic`, scrolling the horizontal axis beyond its document
+bounds only occurs if the document width is greater than the view width,
+or the vertical scroller is hidden and the horizontal scroller is visible.
+* `automatic` - Automatically determine whether to allow elasticity on this axis.
+* `none` - Disallow scrolling beyond document bounds on this axis.
+*`allowed` - Allow content to be scrolled past its bounds on this axis in an elastic fashion.
+
+#### `scrollView.setVerticalScrollElasticity(elasticity)` _macOS_
+
+The scroll view’s vertical scrolling elasticity mode.
+
+#### `scrollView.getHorizontalScrollElasticity()` _macOS_
+
+Returns `string` - The scroll view’s horizontal scrolling elasticity mode.
+
+#### `view.getVerticalScrollElasticity()` _macOS_
+
+Returns `string` - The scroll view’s vertical scrolling elasticity mode.
+
+#### `view.setScrollPosition(point)` _macOS_
+
+* `point` - The point in the `contentView` to scroll to.
+
+Scrolls the view’s closest ancestor `clipView` object so a point in the
+view lies at the origin of the clip view's bounds rectangle.
+
+#### `view.getScrollPosition()` _macOS_
+
+Returns `Point`
+
+#### `view.getMaximumScrollPosition()` _macOS_
+
+Returns `Point`
+
+#### Events
+
+#### `scroll-started`
+#### `scroll-ended`
+#### `bounds-did-change`
+
+
+## WrapperBrowserView
+
+A `WrapperBrowserView` is the wrapper for `BrowserView`. It extends `BaseView`.
+
+Process: **Main**
+
+### `new WrapperBrowserView([options])`
+
+* `options` Object (optional)
+  * `browserView` (optional)
+
+If `browserView` is not set then new `BrowserView` is created.
```

Comment:
```suggestion
If `browserView` is not set then new a `BrowserView` is created.
```

Reply:
fixed

Reply:
fixed in 372905c240d9fc6953d2e7aefca0b1d6fcd2b795